lumicks/pylake/tests/test_imaging_camera/test_image_stack.py

```diff
@@ -990,10 +1010,11 @@ def test_alignment_multistack(rgb_alignment_image_data, gray_alignment_image_dat
     assert full_stack.num_frames == len(ref_ts)
 
 
-def test_frame_timestamp_ranges_include_true():
+@pytest.mark.parametrize("steps", [2, 4])
```

Comment:
why test additionally with `2`? is this just to avoid using hardcoded values in the actual test?